test/miscellaneous/odp_dyn_workers.c

```diff
@@ -1163,6 +1220,9 @@ static odp_bool_t run_global(global_config_t *config)
 				func_ret = false;
 			}
```

Comment:
For uniformity could add `continue` here.

Reply:
V2.